src/pybamm/expression_tree/operations/replace_symbols.py

```diff
@@ -0,0 +1,248 @@
+import pybamm
```

Comment:
`SymbolReplacer` and its test are copied from https://github.com/pybamm-team/pybamm-eis/blob/c88e134b28c50070dbf7b904c020c16b4b73e1d9/pybammeis/utils.py

Reply:
Edit: removed